code/index.html

```diff
@@ -3,28 +3,139 @@
-    <!-- dont forget to add a css file and link it here! -->
+    <title>DJ Clover</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Teko:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="icon" type="image/png" href="images/favicon.png" alt="favicon" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- HEADER -->
+
+    <nav id="navbar">
+      <a href="#about">About </a>
+      <a href="#contact">Contact </a>
+      <a href="#info">Information</a>
+    </nav>
+
+    <!-- HERO VIDEO -->
+
+    <section id="hero">
+      <video loop muted autoplay preload="auto">
+        <source src="videos/djvideo.mp4" type="video/mp4" />
+        Your browser does not support the video tag
+      </video>
+      <div class="heropanel">
+        <h1>
+          <a rel="home">DJ Clover</a>
+        </h1>
+        <h2>Dance the night away</h2>
+      </div>
+    </section>
+
+    <!-- About -->
 
-    <!-- Signup form -->
+    <section id="about" class="about">
+      <div class="container">
+        <img src="images/dj.jpg" alt="photo of dj" />
+        <div class="serviceText">
+          <h2>DJ CLOVER AT YOUR SERVICE</h2>
+          <h3>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum
+          </h3>
+        </div>
+      </div>
+    </section>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!-- FORM -->
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <section id="contact" class="contact">
+      <div class="container">
+        <div class="form">
+          <div class="contact-us">
+            <h3>CONTACT US</h3>
+          </div>
+          <div class="input-fields">
+            <input type="text" class="input" placeholder="Name" required />
+            <input type="text" class="input" placeholder="Company" />
+            <input type="email" class="input" placeholder="Email" required />
+            <input type="number" class="input" placeholder="Phone" required />
+          </div>
+          <div class="msg">
+            <textarea placeholder="Message"></textarea>
+            <div class="btn" onclick="showAlert('Technigo is awesome')">
+              SEND
+            </div>
```

Comment:
This should probably be a `<button>`-tag instead since it is a button that you're trying to create, something to be mindful about!